tests/testthat/testCombine.R

```diff
@@ -0,0 +1,99 @@
+# testCombine.R
+
+context("test for Combine SNV and CNA files")
+
+# ==== BEGIN SETUP AND PREPARE =================================================
+# set up tempdir and tempfiles for output files and to be deleted after tests
+OLOG <- as.character(getOption("rete.logfile"))   # save original logfile name
+logFileName(fPath = tempdir(), setOption = TRUE)  # make tempdir() the log dir
+logName <- unlist(getOption("rete.logfile"))
+if (file.exists(logName)) { file.remove(logName)}
+# ==== END SETUP AND PREPARE ===================================================
+
+test_that("parameter errors are correctly handled", {
+    # try no parameter input
+    expect_error(combineSNV_CNA(writeLog=FALSE),
+                 'argument "fname" is missing errro')
+
+    # try length-zero or NuLL input vector
+    input_vector <- character()
+    expect_error(combineSNV_CNA(input_vector, writeLog=FALSE),
+                 'empty input vector error')
+
+    # try wrong file format (e.g. text file instead of rds dataframe) in input vector
+    input_vector[1] <- "CNA.txt"
+    expect_error(combineSNV_CNA(input_vector, writeLog=FALSE),
+                 '.checkArgs> "fName" mode error')
```

Comment:
you call the argument fname 3 different things in this file. "fname", "fName", and "fnames".